CREDITS.md

```diff
@@ -39,12 +41,14 @@ Contributors to the Firecracker repository:
 - Andrei Casu-Pop <cpo@amazon.com>
 - Andrei Cipu <acipu@amazon.com>
 - Andrei Sandu <sandreim@amazon.com>
+- Andrew Yao <andr3wy@gmail.com>
 - Andrii Radyk <ander.ender@gmail.com>
 - andros21 <andrea.ros21@murena.io>
 - Angus McInnes <angus@amcinnes.info>
 - Arjun Ramachandrula <avrdev77@gmail.com>
 - Arun Gupta <arun.gupta@gmail.com>
 - Arunav Sanyal <Khalian@users.noreply.github.com>
+- Ashwin Ginoria <ec2-user@ip-10-0-10-88.us-west-2.compute.internal>
```

Comment:
Could we fix this one with a `.mailmap` entry?

Reply:
How do I fix this if it is the only entry?

Reply:
1. How did you find the email?
2. Should I add the `Ashwin Ginoria <aginoria@amazon.com>` in CREDITS and `Ashwin Ginoria <ec2-user@ip-10-0-10-88.us-west-2.compute.internal>` in `.mailmap` or smth else?

Reply:
Add the line above line exactly to `.mailmap`, and re-run `update-credits.sh`

Reply:
done